src/features/import/utils/prepareImportOperations.ts

```diff
@@ -63,12 +65,9 @@ export default function prepareImportOperations(
         if (column.kind === ColumnKind.ORGANIZATION) {
           column.mapping.forEach((mappedColumn) => {
             if (mappedColumn.value === row.data[colIdx]) {
-              if (!personImportOps[rowIndex].organizations) {
-                personImportOps[rowIndex].organizations = [];
-              }
-              personImportOps[rowIndex].organizations = personImportOps[
-                rowIndex
-              ].organizations?.concat(mappedColumn.orgId as number);
+              personImportOps[rowIndex].organization = [
+                mappedColumn.orgId as number,
```

Comment:
Why is this not typed as `number` originally?

Reply:
It can be `null` 
![image](https://github.com/zetkin/app.zetkin.org/assets/77925373/f5e96a7d-fc17-442b-8ae6-bbb8dcb4adc0)

Reply:
Ok thanks for explaining. If it can be `null`, the typecast here (`as`) is opening up for bugs. That's why typecasting should always be avoided if possible. If `orgId` can be `null`, and `organization` needs to be an array that never contains `null` values, the solution should not (as here) to tell typescript "ignore this incompatibility, I promise it isn't `null`" but to actually verify that it isn't null using an if statement or something.